tools/botskills/src/botskills-connect.ts

```diff
@@ -31,8 +31,8 @@ program.Command.prototype.unknownOption = (flag: string): void => {
 program
     .name('botskills connect')
     .description('Connect a skill to your assistant bot. Only one of both path or URL to Skill is needed.')
-    .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
-    .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-l, --localManifest <path>', 'Path to local Skills Manifest files. To connect multiple skills, use multiple paths separated by a comma')
```

Comment:
The `<path>` should be `[path]`? I don't know, it's just a suggestion, same for `<url>`. We should review another examples

Also:
```suggestion
    .option('-l, --localManifest <path>', 'Paths to local Skill Manifest file. To connect multiple skills, use multiple paths separated by a comma')
```